pilot/pkg/model/context.go

```diff
@@ -1320,14 +1320,12 @@ func (node *Proxy) EnableHBONE() bool {
 // strategy, depending on the binding of the waypoints to its backend
 // workloads.
 type WaypointScope struct {
-	Namespace      string
-	ServiceAccount string // optional
+	Namespace string
```

Comment:
nit: do we even need this struct anymore? 

It's replaced in most places by `(network, address string)` and it is now a struct around a single string, so I'm guessing we don't.

Reply:
probably not... I originally kept it thinking it would be useful but I think it's vestigial at this point for all intents and purposes

Reply:
I've removed it in the latest commit.